torch/distributions/fishersnedecor.py

```diff
@@ -37,6 +37,18 @@ def __init__(self, df1, df2):
             batch_shape = self.df1.size()
         super(FisherSnedecor, self).__init__(batch_shape)
 
+    @property
+    def mean(self):
+        df2 = self.df2.clone()
+        df2[df2 <= 2] = float('nan')
```

Comment:
Is it `nan` or `inf`? Similarly for `variance`.

Reply:
`nan` is what I inferred from wikipedia. Tbh, that's up to you: you know F distribution the best among us, I'm ok with putting whichever you say. 😄

Reply:
I think that from http://mathworld.wolfram.com/F-Distribution.html, it is `inf`.